common/debug.c

```diff
@@ -92,9 +92,9 @@ bool transaction_cmp(iota_transaction_t *tx1, iota_transaction_t *tx2) {
   bool equal = true;
   field_mask_t old_mask1 = {}, old_mask2 = {};
   memcpy(&old_mask1, &tx1->loaded_columns_mask, sizeof(field_mask_t));
-  memset(&tx1->loaded_columns_mask, 0xFFFFF, sizeof(field_mask_t));
+  memset(&tx1->loaded_columns_mask, 0xFFFF, sizeof(field_mask_t));
```

Comment:
The second parameter will be converted into unsigned char. So, you can use `0xFF` instead.